src/OrchardCore.Modules/OrchardCore.Twitter/Startup.cs

```diff
@@ -47,7 +48,17 @@ public override void ConfigureServices(IServiceCollection services)
 
             services.AddHttpClient<TwitterClient>()
                 .AddHttpMessageHandler<TwitterClientMessageHandler>()
-                .AddTransientHttpErrorPolicy(policy => policy.WaitAndRetryAsync(3, attempt => TimeSpan.FromSeconds(0.5 * attempt)));
+                .AddResilienceHandler("oc-handler", builder => builder
+                    .AddRetry(new HttpRetryStrategyOptions
+                    {
+                        Name = "oc-retry",
```

Comment:
Maybe we should move the HttpRetryStrategyOptions into a static instance so we can reuse it in other places.

Reply:
> so we can reuse it in other places

Which places do you mean?